### PR TITLE
Add organic field

### PR DESCRIPTION
**Pull request by hleroy**
Adding organic field as requested in Issue #4

### CODE REVIEW

File: app/src/main/java/com/myadridev/mypocketcave/activities/BottleDetailActivity.java [hunk 1]
```diff
@@ -173,6 +175,7 @@ private void setLayoutValues() {
             wineColorIconView.setImageDrawable(ContextCompat.getDrawable(this, wineColorDrawableId));
         }
         wineColorView.setText(bottle.WineColor.StringResourceId);
+        organicView.setVisibility(bottle.Organic ? View.VISIBLE : View.INVISIBLE);
```
Comment (adridevparis) [inline app/src/main/java/com/myadridev/mypocketcave/activities/BottleDetailActivity.java]: could be View.GONE

File: app/src/main/res/layout/suggest_bottle_search.xml [hunk 2]
```diff
@@ -255,11 +255,36 @@
                 android:layout_centerHorizontal="true"
                 android:layout_centerVertical="true" />
 
+            <TextView
```
Comment (adridevparis) [inline app/src/main/res/layout/suggest_bottle_search.xml]: I rather thought to a spinner with 3 options :
- Any type of farming
- Organic farming
- Not organic farming

File: app/src/main/java/com/myadridev/mypocketcave/models/v2/SuggestBottleCriteriaV2.java [hunk 3]
```diff
@@ -50,6 +50,8 @@
     public CaveLightModelV2 Cave;
     @SerializedName("icr")
     public boolean IsCaveRequired;
+    @SerializedName("ior")
+    public boolean IsOrganicRequired;
```
Comment (adridevparis) [inline app/src/main/java/com/myadridev/mypocketcave/models/v2/SuggestBottleCriteriaV2.java]: would be another boolean field, for example IsOrganic (Cf. layout)

File: app/src/main/java/com/myadridev/mypocketcave/managers/BottleManager.java [hunk 4]
```diff
@@ -150,6 +150,9 @@ public static int getBottlesCount(Collection<BottleModelV2> bottles, WineColorEn
             if (searchCriteria.IsCaveRequired && caveScore == 0) {
                 continue;
             }
+            if (searchCriteria.IsOrganicRequired && !bottle.Organic) {
```
Comment (adridevparis) [inline app/src/main/java/com/myadridev/mypocketcave/managers/BottleManager.java]: Rather a compute score with another boolean field (Cf. layout)

File: app/src/main/res/layout/suggest_bottle_search.xml (hunk 2 above)
Comment (hleroy) [inline app/src/main/res/layout/suggest_bottle_search.xml]: I initially thought to implement 3 options with a spinner and compute a score (like the other fields). Then I realized that it was unlikely that someone would want to search specifically a non organic farming. To simplify the user experience, I kept just the checkbox. But if you prefer that I implement a spinner, I can do it.

**Pull request closed without merging**